source/_posts/2017/2017-01-10-jak-psat-zajimave-inzeraty-pro-ajtaky.md

```diff
@@ -0,0 +1,183 @@
+ 
+## Jak na to? Není to o vás, je to o tom, kdo hledá
+
+Spousta inzerátu je psáno spíše jako sebeprezentace firmy - co umí, v čem je nejlepší a co dokázala. **Lidi čtou, protože chtějí něco pro sebe** - informace, peníze nebo zábavu. **Co ze článku nebo pracovní pozice budou mít, hledají hned na začátku. Dejte jim to do první třetiny!**
+
+Tip na to, jak lépe psát najdeš tady: 
+[3 minižíky od Otty](http://ottocopy.cz/ebook-zdarma) - hledej sekci *O nás*.
+
+### Buďte upřímní
+
+Uvádějte jen technologie, které opravdu máte. Ne ty, které plánujte nebo chcete, ale ty které opravdu máte. Když vám na pohovor přijde člověk, který se těší na PHP 7, a vy ho budete teprve nasazovat (kdo ví kdy), začne mít pocit, že vaříte z vody.
+
+**Text musí být upřímný, aby čtenář dostal to, co očekává. Jedině tak vám bude věřit a vrátí se k vám.**
+
+### Kamarád taky rád
+
+**Když u vás pracuje někdo šikovný, zmiňte ho v inzerátu**. Lidé půjdou spíše za kamarádem nebo někým v komunitě známým, než jen do firmy.
+
+Sám si pamatuju tyhle dvojice (i když už tam někteří nepracují): 
+
+- Filip Procházka - DámeJídlo, Rohlík 
+- Ondrej Mirtes - Slevomat
+- Patrik Votoček - Shipito
+- Vašek Purchart - Lavito
+
+
+### Řeknete si o feedback - dovnitř i ven
+
+**1. Řekněte si o feedback v práci** 
+
+„Co tě nakonec dovedlo k nám místo ke konkurenci?“
+
+**2. Dejte to přečíst někomu zvenčí**
+
+Vaši zaměstnanci vás přirozeně vidí lépe než někdo, koho neživíte. Lidé zvenčí vám dají upřímnější zpětnou vazbu a spíše řeknou, co potřebují pro navázání vztahu.
+ 
+**3. Které dvě věci se ti nejvíc líbí a které dvě bys zlepšil?**    
+    
+Napište 3 lidem z cílovky. Tak máte šanci se dozvědět, jak působíte navenek &ndash; očima člověka, kterého hledáte.
+
+       
+### Svlékněte se
+
+Ukažte, jak to doopravdy děláte. Důkaz místo slibů.
+
+**Děláte open-source?** Ven s ním. Vím, že Skrz, Jobs, Slevomat, PeckaDesign nebo Shopsys mají balíčky na Githubu. Bohužel, ani u jednoho z inzerátů jsem odkaz nenašel.
+
+Více informací mi může dát jenom náhled do zdrojáků aplikace. **Open-source je velmi podceňovaný hiring nástroj.**
+    
+### Odlište se
+
+Prodejte své odlišnosti. Proto k vám lidi jdou, ne protože stejně jako ostatní děláte v PHP, Symfony, Nette a Doctrine. Taková informace by tam měla být, ale určitě ne na předních místech.
+
+### Příklady přitažlivých odlišností
+
+Znám firmy, které:
+
+- **Každý měsíc mají půldenní školení** - dobrovolné a na přání programátorů.
+
+- Znám taky **„každý měsíc máme půldenní školení“**. To je ale povinné a zadání volí ředitel nebo jiné oddělení. Není větší pain, než být na školení „Jak dělat marketing na Instagramu“, když se chci naučit škálovat výkon přes sharding serverů. 
+
+- **Dávají každému ročně 10 000 Kč na vzdělání** a může si sám vybrat, jak je investuje.
+- **Každý poslední pátek v měsící dávají programátorům možnost refaktorovat** jakoukoliv část kódu chtějí.
+       
```

Comment:
Nevím teď u koho to bylo, ale vývojáři měli možnost 1-2 měsíčně pracovat na vlastním side projektu v libolné technologii, který řeší nějaký problémy firmy.